action.yml

```
@@ -17,7 +17,7 @@ outputs:
 
 runs:
   using: 'docker'
-  image: 'docker://betrybe/jest-evaluator-action:v9.4'
+  image: 'docker://betrybe/jest-evaluator-action:igorg-teste-logs'
```

```suggestion
  image: 'docker://betrybe/jest-evaluator-action:v9.5'
```

Igão, aqui já dá pra deixar o número da versão que você vai lançar

---

Vdd, valeu Camelo!